extension/entitystore/extension.go

```diff
@@ -203,6 +203,7 @@ func (e *EntityStore) shouldReturnEntity() bool {
 		e.stsClient = sts.New(
 			e.nativeCredential,
 			&aws.Config{
+				Region:   aws.String(e.config.Region),
```

Comment:
this may not even be an issue but just thought I would mention. 

`e.config.region` comes from default EC2 region. But if customers configures the log agent credential to use a different account in a different region(i think this is possible?), there will be a mismatch in the region used for `e.nativeCredential` and the region used in this line.

Reply:
Is there a way to get the customer configured region here? So when we do the config translation, we don't feed in the customer configuration to the entity store?